### ISSUE TITLE
Netty5 Documentation: dead link to io.netty5.buffer.ByteBuf, which does not exist

### ISSUE THREAD

**Issue by Stewori** [netty/netty#14062]
This is just a documentation issue. In https://netty.io/5.0/api/io.netty5.codec/module-summary.html the doc mentions the non-existing class `io.netty5.buffer.ByteBuf` (such a class existed in netty 4).

**Comment by chrisvest** [comment]
Yeah, these tend to sneak in when we forward merge. Looks like that summary page hasn't been updated at all, though.
